### ISSUE TITLE
VB -> C#: uses var instead of object when DirectCasting ?

### ISSUE THREAD

**Issue by v1nce** [icsharpcode/CodeConverter#400]
DirectCast in VB is converted to object. Maybe it would be better to go for var instead.
Then the convertor mismatches fun call vs array-like accessing  

### Input code
```vbnet
Dim row = DirectCast(item, DataRowView).Row
Dim valueItem As String = row(foo.ValueMember).ToString()
```

### Erroneous output
```C#
object row = ((DataRowView)item).Row;
string valueItem = row(foo.ValueMember).ToString();
```

### Expected output
```C#
DataRow row = ((DataRowView)item).Row;
or
var row = ((DataRowView)item).Row;
string valueItem = row[foo.ValueMember].ToString();
```

### Details
* Product in use:  VS extension
* Version in use: 7.2.0.0

**Comment by GrahamTheCoder** [comment]
Both those issues sound like the converter can't figure out the type of DataRowView. The first place to look would be at the import statements and references (for the Web converter it's possible this isn't in the list of references it guesses). Thanks for raising the issue!

**Comment by GrahamTheCoder** [comment]
This was fixed in the 7.3 release as part of another issue
